src/platform_impl/linux/event_loop.rs

```diff
@@ -462,35 +468,34 @@ impl<T: 'static> EventLoop<T> {
               glib::Propagation::Proceed
             });
             window.connect_button_press_event(move |window, event| {
-              if !window.is_decorated()
+              const LMB: u32 = 1;
+              if (is_wayland && window.is_decorated())
```

Comment:
This should be `||` not `&&` and should be `!window.is_decorated()` because we want to always keep this handling on wayland (because it doesn't have server side decorations) but if not wayland we only handle the non decorated windows

```suggestion
              if (is_wayland || !window.is_decorated())
```

Reply:
OMG, I’ll change it right now.